File: 1491. Average Salary Excluding the Minimum and Maximum Salary/solution.py
class Solution(object):
  def average(self, salary):
    """
    :type salary: List[int]
    :rtype: float
    """
    minimum = salary[0]
    maximum = salary[0]
    total = 0.00
    for employee in salary:
      total += employee
      if employee > maximum:
        maximum = employee
      if employee < minimum:
        minimum = employee
    
    total -= minimum + maximum
    return total / (len(salary) - 2)

File: 1491. Average Salary Excluding the Minimum and Maximum Salary/solution.cpp
class Solution {
public:
  double average(vector<int>& salary) {
    int minimum = salary[0];
    int maximum = salary[0];
    double total = 0.00;
    for (int index = 0; index < salary.size(); index++){
      int employee = salary[index];
      total += employee;
      if (employee > maximum){
        maximum = employee;
      }
      if (employee < minimum){
        minimum = employee;
      }
    }

    total -= minimum + maximum;
    return total / (salary.size() - 2);
  }
};